src/PlaybackState.js

```diff
@@ -0,0 +1,10 @@
+export default {
+  // The player is actively playing audio.
+  Playing: 'PLAYING',
+
+  // The player has audio available to play, but has been set to a paused state.
+  Paused: 'PAUSED',
+
+  // The player has no available audio to play.
+  Stopped: 'STOPPED',
+};
```

Comment:
Curious why these attributes are capitalized instead of all lowercase or all uppercase. I'm used to seeing "constants" as all uppercase.